service-tests/npm.js

```diff
@@ -26,3 +28,56 @@ t.create('gets the tagged package version of @cycle/core')
 t.create('invalid package name')
   .get('/v/frodo-is-not-a-package.json')
   .expectJSON({ name: 'npm', value: 'invalid' });
+
+t.create('public domain license')
```

Comment:
I'm creating tests for current implementation, then I will change production code.

Reply:
done